src/app/components/Companies.js

```diff
@@ -1,31 +1,69 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import getCompaniesData from '../store/actions/getCompaniesData';
+import Grid from '@material-ui/core/Grid';
+import Divider from '@material-ui/core/Divider';
+import Chip from '@material-ui/core/Chip';
+import Link from '@material-ui/core/Link';
+
+// Styles
+const centerText = { textAlign: 'center' };
+const tagStyle = { height: '25px', marginRight: '5px', backgroundColor: '#65B8DE' };
+
+// Helper function
+const chooseRandomTags = (tagsArray, n) => {
+  // Shuffle array
+  const shuffled = tagsArray.sort(() => 0.5 - Math.random());
+
+  // Get sub-array of first n elements after shuffled
+  const selected = shuffled.slice(0, n);
+
+  return selected;
+};
 
 class Companies extends React.Component {
   componentDidMount() {
-    const { getCompaniesData } = this.props; // eslint-disable-line
+    const { getCompaniesData } = this.props;
     getCompaniesData();
   }
 
   renderCompaniesList() {
-    const { companies } = this.props; // eslint-disable-line
+    const { companies } = this.props;
     return companies.map((company, index) => {
       return (
-        // placeholder for company cards
-        <div key={index}>
-          <h4>{company.name}</h4>
+        <div key={index} style={{ height: '100px' }}>
+          <h4 style={{ ...centerText, margin: '0 0 20px' }}>
+            <Link href={'/companies/' + company.id} color="inherit">
```

Comment:
Likewise this should be `<Link to=/blahblahblah>`, see Header.js for reference :)